pkg/timelock/operations.go

```diff
@@ -51,13 +52,26 @@ func (tw *Worker) executeCallSchedule(ctx context.Context, c *contract.TimelockT
 		})
 	}
 
+	chainID, err := tw.ethClient.NetworkID(context.Background())
+	if err != nil {
+		return nil, err
+	}
+
+	txOpts := &bind.TransactOpts{
+		From:    fromAddress,
+		Signer:  tw.signTx,
+		Context: ctx,
+	}
+
+	// if chainId is zksync-testnet or mainnet use custom gasPrice to enforce legacy tx
+	if chainID.Cmp(big.NewInt(300)) == 0 || chainID.Cmp(big.NewInt(324)) == 0 {
```

Comment:
Let's avoid magic numbers. Can we pull in the chain-selectors repo or define these as top level variables?

Reply:
added!